src/main/java/Chronos.java

```diff
@@ -0,0 +1,355 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
```

Comment:
Are these import statement following the right order?
Maybe consider follow by the order of lexicographical.